src/components/Login/styles.module.scss

```
@@ -0,0 +1,65 @@
+.container {
+  position: relative;
+  height: 100vh;
+  
+  .loginContainer {
+    background-color: #fff;
+    margin: auto;
+    width: 350px;
```

para que esto sea completamente responsive debería tener:

```suggestion
    width: 100%;
    max-width: 350px;
```